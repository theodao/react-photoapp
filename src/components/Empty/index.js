import React from 'react';
import { EmptyState } from '@gotitinc/design-system';

export default ({ title = '' }) => (
  <div
    style={{
      maxWidth: '1140px',
      display: 'flex',
      height: '100%',
      alignItems: 'center',
      justifyContent: 'center',
      margin: '0 auto',
    }}
  >
    <EmptyState name="general">
      <EmptyState.Heading>{title} Empty</EmptyState.Heading>
    </EmptyState>
  </div>
);
