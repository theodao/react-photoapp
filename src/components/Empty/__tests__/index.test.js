import React from 'react';
import { cleanup, render } from '@testing-library/react';
import Empty from '../';

describe('Testing empty component', () => {
  afterEach(cleanup);

  it('Should match snapshot', () => {
    const { asFragment } = render(<Empty />);
    expect(asFragment()).toMatchSnapshot();
  });
});
