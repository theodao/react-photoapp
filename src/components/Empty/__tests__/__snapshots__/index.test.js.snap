// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing empty component Should match snapshot 1`] = `
<DocumentFragment>
  <div
    style="max-width: 1140px; display: flex; justify-content: center; margin: 0px auto;"
  >
    <div
      class="EmptyState u-inlineBlock u-textCenter"
    >
      <img
        alt="General"
        class="u-maxWidthFull"
        src="https://designsystem-cdn.got-it.ai/images/general.8da83c28.svg"
        width="240"
      />
      <div
        class="EmptyState-heading u-marginTopSmall u-text650 u-textLight"
      >
         Empty
      </div>
    </div>
  </div>
</DocumentFragment>
`;
